style.css

```diff
@@ -312,6 +330,13 @@ main {
   border-radius: 4px;
 }
 
```

Comment:
- [ ] You did a great job setting up the HTML validation and integrating the Formspree service. :+1:  Let's work on a minor detail to match the Figma design. Could you add the background image as shown in the design template? 🧐️ Please ensure the background is shown on devices with a minimum width of at least 375 px.


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://github.com/Young-Gaius/portfolio-setup/assets/17243325/40920869-bd32-4df0-8bb9-f692b674a37c" width="300">|<img src="https://github.com/Young-Gaius/portfolio-setup/assets/17243325/7fc8c4a2-c7b2-44e1-bd71-970a2e16c5ba" width="300">|